tools/code_format/config.yaml

```diff
@@ -326,3 +326,6 @@ visibility_excludes:
 - source/extensions/quic/server_preferred_address/BUILD
 - source/extensions/listener_managers/listener_manager/BUILD
 - source/extensions/upstreams/tcp/BUILD
+- source/extensions/health_checkers/BUILD
```

Comment:
Note: Intentionally keeping Envoy's redundant lines (https://github.com/envoyproxy/envoy/blob/d32189df676e84be7d0328dcaf6df889e11ceb6e/tools/code_format/config.yaml#L315-L317) so it doesn't show up as a Nighthawk-specific customization.